Index: events.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[\r\n  {\r\n    \"id\": 1,\r\n    \"original_id\": 1757946651231456000,\r\n    \"chat_id\": 666460598,\r\n    \"text\": \"text\",\r\n    \"datetime\": \"2025-09-15T17:32:00+03:00\"\r\n  },\r\n  {\r\n    \"id\": 2,\r\n    \"original_id\": 1757946640220276300,\r\n    \"chat_id\": 666460598,\r\n    \"text\": \"text\",\r\n    \"datetime\": \"2025-09-15T17:34:00+03:00\"\r\n  },\r\n  {\r\n    \"id\": 3,\r\n    \"original_id\": 1757946656905489100,\r\n    \"chat_id\": 666460598,\r\n    \"text\": \"text\",\r\n    \"datetime\": \"2025-09-15T17:35:00+03:00\"\r\n  }\r\n]
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/events.json b/events.json
--- a/events.json	(revision 37a80a937cac757bccc705ce8278d4596d2041c4)
+++ b/events.json	(date 1758143367615)
@@ -1,23 +1,1 @@
-[
-  {
-    "id": 1,
-    "original_id": 1757946651231456000,
-    "chat_id": 666460598,
-    "text": "text",
-    "datetime": "2025-09-15T17:32:00+03:00"
-  },
-  {
-    "id": 2,
-    "original_id": 1757946640220276300,
-    "chat_id": 666460598,
-    "text": "text",
-    "datetime": "2025-09-15T17:34:00+03:00"
-  },
-  {
-    "id": 3,
-    "original_id": 1757946656905489100,
-    "chat_id": 666460598,
-    "text": "text",
-    "datetime": "2025-09-15T17:35:00+03:00"
-  }
-]
\ No newline at end of file
+[]
\ No newline at end of file
